ElementX/Sources/Screens/SecureBackup/SecureBackupScreen/View/SecureBackupScreen.swift

```diff
@@ -76,10 +77,7 @@ struct SecureBackupScreen: View {
                 context.send(viewAction: .keyBackup)
             })
         default:
-            ListRow(kind: .custom {
-                ProgressView()
-                    .frame(maxWidth: .infinity)
-            })
+            ListRow(label: .plain(title: L10n.commonLoading), details: .isWaiting(true), kind: .button { })
```

Comment:
Applies to the others too :)
```suggestion
            ListRow(label: .plain(title: L10n.commonLoading), details: .isWaiting(true), kind: .label)
```